app/models/menu_recipe.rb

```diff
@@ -0,0 +1,26 @@
+class MenuRecipe < ApplicationRecord
+    belongs_to :recipe
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.